autoremovetorrents/condition/freespace.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding:utf-8 -*-
+
+from .sortbase import ConditionWithSort
+import psutil
+
+
+class FreeSpaceCondition(ConditionWithSort):
+    def __init__(self, settings):
+        ConditionWithSort.__init__(self, settings['action'])
+        self._min = settings['min'] * 1073741824  # limit = limit * 1GiB
+        self._path = settings['path']
+
+    def apply(self, torrents):
+        torrents = list(torrents)
+        ConditionWithSort.sort_torrents(self, torrents)
+        torrents = reversed(torrents)
+        _, _, free_space = psutil.disk_usage(self._path)
```

Comment:
Actually `psutil.disk_usage` has 4 return values: total, used, free and percent. The last return value should be represented by `_` too otherwise we will receive a `ValueError` at runtime.